gammapy/catalog/fermi.py

```diff
@@ -185,6 +186,22 @@ def _info_lightcurve(self):
     def is_pointlike(self):
         return self.data["Extended_Source_Name"].strip() == ""
 
+    def _set_position_error(self, model):
+        d = self.data
+        if "Conf_68_PosAng" in d:
+            percent="68"
```

Comment:
As discussed on Slack, this is not correct.
You should implement the inverse of the formula here:
https://github.com/gammapy/gammapy/blob/35d52370f283ad250d17bfc6af0803247afa3b1f/gammapy/utils/gauss.py#L88
and then call it like this:
```
sigma = Gauss2D.from_containment(0.68).sigma
```

It's the catalog source class responsibility to go to a uniform format when creating the model object.
So if 3FHL requires completely different code or the if / else becomes too complex, then you have to put a method in the 3FHL class, even if it means some code duplication.
It's always possible to have a helper method on the base class or anywhere and call that from multiple places.